import-export-cli/integration/throttlePolicy_test.go

```diff
@@ -455,3 +456,31 @@ func TestExportImportThrottlePolicyWithoutTypeFlag(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Shall we add a test to delete a policy that does not exist as well? (Error scenario)

Reply:
Fixed by [859ba91](https://github.com/wso2/product-apim-tooling/pull/928/commits/859ba919a58e2b9a690c34ebf23a137abba6f93b)